meta-sokol-flex-common/classes/mirrors.bbclass

```diff
@@ -2,11 +2,11 @@
 # SPDX-License-Identifier: MIT
 # ---------------------------------------------------------------------------------------------------------------------
 
-PREMIRRORS:append = " \
+PREMIRRORS:append:sokol-flex = " \
 git://sources.redhat.com/ git://sourceware.org/ \n\
 "
 
-MIRRORS += "\
+MIRRORS:sokol-flex += "\
```

Comment:
Same issue as finch, append is being converted to appending to an override which replaces the original rather than appending to it.